src/postformatting/index.js

```diff
@@ -1,12 +1,11 @@
-import debug from 'debug'
-
 import commentRemover from './comment-remover'
 import embeddedbase64Remover from './embedded-base64-remover'
 import unusedKeyframeRemover from './unused-keyframe-remover'
 import unusedFontFaceRemover from './unused-fontface-remover'
 import unwantedPropertiesRemover from './unwanted-properties-remover'
 import ruleSelectorRemover from './rule-selector-remover'
 import finalRuleRemover from './final-rule-remover'
+import debug from 'debug'
```

Comment:
not so important but I prefer external imports at the top of the file.

Reply:
I didn't know the logic of import order. I decided to keep the same order across modules, in most modules it was last one.